apps/print/frontend/src/screens/print_screen.tsx

```diff
@@ -0,0 +1,246 @@
+          {showSplitSelection && (
+            <FormSection style={{ flex: 1 }}>
+              <strong style={{ marginBottom: '0.25rem' }}>Split</strong>
+              <ExpandedSelect
+                selectedValue={selectedSplitName}
+                options={availableSplits
+                  .map((split) => split.name)
+                  .filter(
+                    (name) =>
+                      !searchValue ||
+                      name.toLowerCase().includes(searchValue.toLowerCase())
+                  )}
+                onSelect={(value) => {
+                  setSelectedSplitName(value);
+                }}
+              />
+            </FormSection>
+          )}
+        </Column>
+        <Column>
+          <FormSection>
+            <strong>Party</strong>
+            <RadioGroup
+              label="Party"
+              value={selectedParty}
+              options={parties.map((party) => ({ label: party, value: party }))}
+              onChange={(value: string) => setSelectedParty(value)}
+              hideLabel
+            />
+          </FormSection>
+          <FormSection>
+            <strong>Language</strong>
+            <RadioGroup
+              label="Language"
+              value={selectedLanguage}
+              options={languages.map((language) => ({
+                label: language,
+                value: language,
+              }))}
+              onChange={(value: string) => {
+                setSelectedLanguage(value);
+              }}
+              hideLabel
+            />
+          </FormSection>
+        </Column>
+      </Form>
+      <Footer>
+        {isElectionManagerAuth && (
+          <FooterSection style={{ marginRight: 'auto' }}>
+            <strong>Ballot Type:</strong>
+            <SegmentedButton
+              label="Precinct or Absentee"
+              selectedOptionId={isAbsentee ? 'absentee' : 'precinct'}
+              options={[
+                { label: 'Precinct', id: 'precinct' },
+                { label: 'Absentee', id: 'absentee' },
+              ]}
+              onChange={(newValue) => {
+                setIsAbsentee(newValue === 'absentee');
+              }}
+              hideLabel
+            />
+          </FooterSection>
+        )}
+        <FooterSection>
+          <strong>Copies:</strong>
+          <NumberInput
+            value={numCopies}
+            onChange={(value) => setNumCopies(value || 0)}
+          />
+        </FooterSection>
+        <PrintButton
+          icon="Print"
+          color="primary"
+          fill="filled"
+          onPress={() =>
+            console.log(
+              `Printing ballot style: ${selectedPrecinctName}, ${selectedParty}, ${selectedLanguage}${
+                selectedSplitName ? `, ${selectedSplitName}` : ''
+              }`
+            )
+          }
+          disabled={
+            !selectedPrecinctName ||
+            !selectedLanguage ||
+            !selectedParty ||
```

Comment:
I imagine that, once we wire things up with the backend, party will no longer be required since general elections don't have partisan ballots